plugins/home/src/homePageComponents/QuickStart/Content.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Link } from '@backstage/core-components';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import { ContentModal } from './ContentModal';
+import { useStyles } from './styles';
+
+/**
+ * Props customizing the <QuickStartCard/> component.
+ *
+ * @public
+ */
+export type QuickStartCardProps = {
+  /** The modal link title */
+  modalTitle?: string | React.JSX.Element;
+  /** The link to docs title */
+  docsLinkTitle?: string;
+  /** The link to docs */
+  docsLink?: string;
+  /** The video to play on the card */
+  video?: React.JSX.Element;
+  /** A quickstart image to display on the card */
+  image: React.JSX.Element;
+  /** The card description*/
+  cardDescription?: string;
+  /** A component used to download a quickStart image*/
+  downloadImage?: React.JSX.Element;
+};
+
+/**
+ * A component to display Quick Start info on the homepage.
+ *
+ * @public
+ */
+export const Content = (props: QuickStartCardProps): JSX.Element => {
+  const styles = useStyles();
+  return (
+    <>
+      <ContentModal
+        modalContent={props.image}
+        linkContent={props.modalTitle || 'Onboarding'}
+      />
+      <Typography variant="body1" paragraph>
+        {props.cardDescription || "Get started with Backstage's Catalog"}
```

Comment:
```suggestion
        {props.cardDescription || "Get started with Backstage"}
```

What do you think about using this as default instead? just thinking that its the home page and not specifically the catalog page so might be valuable to keep it more generic